### PR TITLE
Fix some bootstrap 5 left after migration from bootstrap 4

### PR DESCRIPTION
**Pull request by CrestApps**
There are could missed classes after migrating BS 4 to 5.

Also, slightly improve the Card shape to look like this. Red arrow point to the minor changes

image

### CODE REVIEW

**Comment by CrestApps** [comment]
@sebastienros @kevinchalet can these changes be part of the 1.4 release please?

**Comment by hishamco** [comment]
> Also, slightly improve the Card shape to look like this. Red arrow point to the minor changes

@CrestApps how it was before?

File: src/OrchardCore.Themes/TheAdmin/Views/Message.cshtml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 @{
+    if (Model.Type == null)
+    {
+        return;
+    }
```
Comment (Skrypt) [inline src/OrchardCore.Themes/TheAdmin/Views/Message.cshtml:5]: This is not a bootstrap related change. Propose this in a different Pull Request..

Comment (CrestApps) [reply src/OrchardCore.Themes/TheAdmin/Views/Message.cshtml]: done.

File: src/OrchardCore/OrchardCore.DisplayManagement/Shapes/GroupShapes.cs [hunk 2]
```diff
@@ -81,29 +81,38 @@ public async Task<IHtmlContent> Card(IDisplayHelper displayAsync, GroupingViewMo
...
             buttonTag.Attributes["type"] = "button";
             buttonTag.Attributes["data-bs-toggle"] = "collapse";
             buttonTag.Attributes["data-bs-target"] = $"#collapse-{cardIdPrefix}";
             buttonTag.Attributes["aria-expanded"] = "true";
             buttonTag.Attributes["aria-controls"] = $"collapse-{cardIdPrefix}";
 
-            buttonTag.InnerHtml.Append(shape.Grouping.Key);
-            headerTag.InnerHtml.AppendHtml(buttonTag);
+            var buttonIconTag = new TagBuilder("i");
+            buttonIconTag.AddCssClass("fa-solid fa-angle-down");
```
Comment (hishamco) [inline src/OrchardCore/OrchardCore.DisplayManagement/Shapes/GroupShapes.cs:103]: Could we remove the extra lines here?

**Review by hishamco — APPROVED** [review]

**Pull request merged**